optaweb/image.yaml

```diff
@@ -0,0 +1,44 @@
+schema_version: 1
+
+name: "rhdm-7/rhdm70-optaweb"
```

Comment:
This won't be a unique name.
Now we have optaweb-employee-rostering, but soon we'll also build optaweb-vehicle-routing, which is a different end-user template that focusses on a completely different business sector.
I suggest renaming this to "rhdm-7/rhdm70-optaweb-employee-rostering"

Reply:
Adjust also "value: "rhdm-7-rhdm70-optaweb-container"" etc lower down

Reply:
changed